apps/console/src/index.jsp

```diff
@@ -316,7 +316,15 @@
                 // Redirect user to the login page if the prompt parameter is set to login.
                 if (promptParam && promptParam === 'login') {
                     auth.initialize(authConfig);
-                    auth.signIn({ prompt: "login" });
+
+                    const authParams = { prompt: "login" };
+
+                    if (getOrganizationPath()) {
```

Comment:
Need to do the same for static preauth as well.

[Note] This only handles unauthenticated user login scenarios.